.prettierignore

```diff
@@ -0,0 +1,10 @@
+OverlayPlugin.sln/
+OverlayPlugin/
+OverlayPlugin.Common/
+OverlayPlugin.Core/
+OverlayPlugin.Updater/
+Thirdparty/
+dist/
+docs/
+**/*.md
+translations.json
```

Comment:
```suggestion
*
!*.ts
!*/
```

Otherwise, this is what I got when running it:
```
$ npm run format

> overlay-plugin@0.0.0 format
> prettier -w ./

.github\matchers\msbuild.json 87ms
.github\workflows\build-artifact.yml 83ms
.github\workflows\release.yml 19ms
.github\workflows\ts-test.yaml 17ms
AddonExample\obj\AddonExample.csproj.nuget.dgspec.json 69ms
AddonExample\obj\project.assets.json 198ms
HtmlRenderer\obj\HtmlRenderer.csproj.nuget.dgspec.json 8ms
HtmlRenderer\obj\project.assets.json 58ms
lib\index.js 26ms
out\Debug\addons\vk_swiftshader_icd.json 3ms
out\Debug\libs\resources\enmity_target_funcs.js 141ms
out\Debug\libs\resources\enmity\aggrolist.html 198ms
out\Debug\libs\resources\enmity\css\aggrolist.css 88ms
out\Debug\libs\resources\enmity\css\enmity.css 61ms
out\Debug\libs\resources\enmity\css\targetinfo.css 45ms
out\Debug\libs\resources\enmity\enmity_absolute_show_only_monsters.html 23ms
out\Debug\libs\resources\enmity\enmity_absolute.html 36ms
out\Debug\libs\resources\enmity\enmity_relative_show_only_monsters.html 35ms
out\Debug\libs\resources\enmity\enmity_relative_simple.html 27ms
out\Debug\libs\resources\enmity\enmity_relative.html 22ms
out\Debug\libs\resources\enmity\js\aggrolist.js 31ms
out\Debug\libs\resources\enmity\js\enmity.js 42ms
out\Debug\libs\resources\enmity\js\funcs.js 49ms
out\Debug\libs\resources\enmity\js\targetinfo.js 37ms
out\Debug\libs\resources\enmity\targetinfo_mini.html 29ms
out\Debug\libs\resources\enmity\targetinfo.html 18ms
out\Debug\libs\resources\label.html 33ms
out\Debug\libs\resources\loading.html 16ms
out\Debug\libs\resources\miniparse.html 150ms
out\Debug\libs\resources\presets.json 32ms
out\Debug\libs\resources\preview.json 81ms
out\Debug\libs\resources\reserved_log_lines.json 4ms
out\Debug\libs\resources\spelltimer.html 145ms
out\Debug\libs\resources\targetbars\focus_target.html 8ms
out\Debug\libs\resources\targetbars\hover_target.html 4ms
out\Debug\libs\resources\targetbars\target_of_target.html 4ms
out\Debug\libs\resources\targetbars\target.html 5ms
out\Debug\libs\resources\targetbars\targetbars.css 22ms
out\Debug\libs\resources\targetbars\targetbars.js 183ms
out\Debug\libs\x64\vk_swiftshader_icd.json 6ms
out\Debug\libs\x86\vk_swiftshader_icd.json 3ms
out\Debug\resources\enmity_target_funcs.js 19ms
out\Debug\resources\enmity\aggrolist.html 53ms
out\Debug\resources\enmity\css\aggrolist.css 39ms
out\Debug\resources\enmity\css\enmity.css 56ms
out\Debug\resources\enmity\css\targetinfo.css 40ms
out\Debug\resources\enmity\enmity_absolute_show_only_monsters.html 16ms
out\Debug\resources\enmity\enmity_absolute.html 8ms
out\Debug\resources\enmity\enmity_relative_show_only_monsters.html 16ms
out\Debug\resources\enmity\enmity_relative_simple.html 5ms
out\Debug\resources\enmity\enmity_relative.html 16ms
out\Debug\resources\enmity\js\aggrolist.js 8ms
out\Debug\resources\enmity\js\enmity.js 23ms
out\Debug\resources\enmity\js\funcs.js 20ms
out\Debug\resources\enmity\js\targetinfo.js 37ms
out\Debug\resources\enmity\targetinfo_mini.html 19ms
out\Debug\resources\enmity\targetinfo.html 7ms
out\Debug\resources\label.html 21ms
out\Debug\resources\loading.html 8ms
out\Debug\resources\miniparse_debug.html 25ms
out\Debug\resources\miniparse.html 110ms
out\Debug\resources\opcodes.jsonc 19ms
out\Debug\resources\presets.json 13ms
out\Debug\resources\preview.json 74ms
out\Debug\resources\reserved_log_lines.json 8ms
out\Debug\resources\spelltimer.html 93ms
out\Debug\resources\targetbars\focus_target.html 5ms
out\Debug\resources\targetbars\hover_target.html 4ms
out\Debug\resources\targetbars\target_of_target.html 3ms
out\Debug\resources\targetbars\target.html 2ms
out\Debug\resources\targetbars\targetbars.css 32ms
out\Debug\resources\targetbars\targetbars.js 150ms
out\Release\libs\resources\enmity_target_funcs.js 20ms
out\Release\libs\resources\enmity\aggrolist.html 29ms
out\Release\libs\resources\enmity\css\aggrolist.css 40ms
out\Release\libs\resources\enmity\css\enmity.css 34ms
out\Release\libs\resources\enmity\css\targetinfo.css 131ms
out\Release\libs\resources\enmity\enmity_absolute_show_only_monsters.html 17ms
out\Release\libs\resources\enmity\enmity_absolute.html 10ms
out\Release\libs\resources\enmity\enmity_relative_show_only_monsters.html 19ms
out\Release\libs\resources\enmity\enmity_relative_simple.html 9ms
out\Release\libs\resources\enmity\enmity_relative.html 8ms
out\Release\libs\resources\enmity\js\aggrolist.js 9ms
out\Release\libs\resources\enmity\js\enmity.js 13ms
out\Release\libs\resources\enmity\js\funcs.js 20ms
out\Release\libs\resources\enmity\js\targetinfo.js 6ms
out\Release\libs\resources\enmity\targetinfo_mini.html 11ms
out\Release\libs\resources\enmity\targetinfo.html 8ms
out\Release\libs\resources\label.html 16ms
out\Release\libs\resources\loading.html 7ms
out\Release\libs\resources\miniparse.html 98ms
out\Release\libs\resources\opcodes.jsonc 6ms
out\Release\libs\resources\presets.json 9ms
out\Release\libs\resources\preview.json 25ms
out\Release\libs\resources\reserved_log_lines.json 4ms
out\Release\libs\resources\spelltimer.html 66ms
out\Release\libs\resources\targetbars\focus_target.html 4ms
out\Release\libs\resources\targetbars\hover_target.html 3ms
out\Release\libs\resources\targetbars\target_of_target.html 5ms
out\Release\libs\resources\targetbars\target.html 5ms
out\Release\libs\resources\targetbars\targetbars.css 11ms
out\Release\libs\resources\targetbars\targetbars.js 106ms
out\Release\libs\x64\vk_swiftshader_icd.json 2ms
out\Release\libs\x86\vk_swiftshader_icd.json 2ms
out\Release\resources\enmity_target_funcs.js 18ms
out\Release\resources\enmity\aggrolist.html 26ms
out\Release\resources\enmity\css\aggrolist.css 24ms
out\Release\resources\enmity\css\enmity.css 47ms
out\Release\resources\enmity\css\targetinfo.css 25ms
out\Release\resources\enmity\enmity_absolute_show_only_monsters.html 13ms
out\Release\resources\enmity\enmity_absolute.html 6ms
out\Release\resources\enmity\enmity_relative_show_only_monsters.html 9ms
out\Release\resources\enmity\enmity_relative_simple.html 3ms
out\Release\resources\enmity\enmity_relative.html 8ms
out\Release\resources\enmity\js\aggrolist.js 11ms
out\Release\resources\enmity\js\enmity.js 10ms
out\Release\resources\enmity\js\funcs.js 18ms
out\Release\resources\enmity\js\targetinfo.js 10ms
out\Release\resources\enmity\targetinfo_mini.html 11ms
out\Release\resources\enmity\targetinfo.html 4ms
out\Release\resources\label.html 19ms
out\Release\resources\loading.html 4ms
out\Release\resources\miniparse_debug.html 9ms
out\Release\resources\miniparse.html 59ms
out\Release\resources\opcodes.jsonc 2ms
out\Release\resources\presets.json 6ms
out\Release\resources\preview.json 26ms
out\Release\resources\reserved_log_lines.json 3ms
out\Release\resources\spelltimer.html 67ms
out\Release\resources\targetbars\focus_target.html 3ms
out\Release\resources\targetbars\hover_target.html 3ms
out\Release\resources\targetbars\target_of_target.html 4ms
out\Release\resources\targetbars\target.html 7ms
out\Release\resources\targetbars\targetbars.css 20ms
out\Release\resources\targetbars\targetbars.js 111ms
package-lock.json 6ms
package.json 2ms
test\global-ref.d.ts 639ms
test\global.ts 11ms
test\test.ts 4ms
tsconfig.json 12ms
types\index.d.ts 49ms
UpgradeLog.htm 4ms
```

I'm not opposed to formatting json/html/css etc with prettier, but maybe that'd be good to enable in a future PR.

Reply:
I didn't build the plugin, didn't know there are js/json/html files in these dir, I'll add them to ignore file.